src/idiomatic/leveraging-the-type-system/extension-traits/method-resolution-conflicts.md

```diff
@@ -0,0 +1,79 @@
+---
+minutes: 15
+---
+
+# Method Resolution Conflicts
+
+What happens when you have a name conflict between an inherent method and an
+extension method?
```

Comment:
I think this content is missing transition. Creating naming conflicts is (obviously?) a bad idea, so how did we end up with code like this? Instead of figuring out what happens, shouldn't we remember simpler advice "don't do this"?

One way to introduce this slide is to set up a situation where we have added an extension method without conflicts, but then the type (owned by another crate) adds an inherent method.

Reply:
Ah sorry I missed that while writing this comment!